src/components/PlayerView/playerViewConfig.ts

```diff
@@ -19,6 +19,13 @@ export interface PlayerViewConfig {
    * Provides options to configure Picture in Picture playback.
    */
   pictureInPictureConfig?: PictureInPictureConfig;
+
+  /**
+   * When set to `true`, the first frame of the main content will not be rendered before playback starts. Default is `false`.
```

Comment:
Please mention that this will have no effect with `UserInterfaceType.Subtitle`, at least on iOS.

Reply:
Added in d63c39cd.

Android doesn't mention any restrictions etc in the API doc, plus for Android it's on the `PlayerViewConfig`, so I assume no such limitation exists there. But would be good to get confirmation from the Android team before marking this conversation as resolved.

Reply:
@matamegger could you share some insights?

Reply:
The property should work for all UI types of the android player :)

Reply:
Actualy here is no good reason to not have it on iOS, I think it was just never requested.
@dweinber It would be a quite simple change to support it if needed.

Reply:
Good to know, not needed at the moment but would be nice to have 🙂